add-ons/ansible-service-broker/README.md

```diff
@@ -15,25 +15,20 @@ $ minishift config set openshift-version v3.7.1
 
 Minishift must be deployed with the Service Catalog enabled
 
+**NOTE**: Prior to minishift `v1.16.1`, minishift was installed with the Service
+Catalog in a different way. These instructions are for minishift `>=v1.16.1`,
```

Comment:
It actually doesn't matter and would have worked for version prior to v1.16.0 as well... (only v1.16.0 has an issue).